build.gradle

```diff
@@ -1,10 +1,11 @@
 plugins {
-  id 'net.researchgate.release' version '2.4.1'
+  id 'net.researchgate.release' version '2.7.0'
   id "com.jfrog.bintray" version "1.7"
 }
 
 apply plugin: 'java'
 apply from: 'quality.gradle'
+apply from: 'release.gradle'
```

Comment:
will it be possible to utilize release scripts from there:
https://github.com/reportportal/gradle-scripts?

Reply:
do we have some working example for injecting these scripts? the only thing that comes to my mind is to write/use some HTTP resource loader on groovy to import that from URL and execute (eval?) as a script. I'm really newbie in gradle though, so it might be something obvious that I don't know

Reply:
nevermind, it's simple indeed, found here:
https://github.com/reportportal/service-rally/blob/51191fd0825c152f16a3d4985247418f82ec4cba/build.gradle

Reply:
ok, just checked this script's content on my local repo: extra config doesn't harm and release is executed successfully + auto published.
https://travis-ci.org/andreifinski/agent-java-spock/jobs/457064940